src/index.ts

```diff
@@ -54,8 +55,8 @@ export default class IntegrationTestEnvironment extends NodeEnvironment {
     }
 
     const ipfs = await buildIpfs(config.jest.services.ipfs)
-    const ceramic = await buildCeramic(config.jest.services.ceramic, ipfs)
-    const ceramicClient = await buildCeramic(config.jest.services.ceramicClient)
+    const ceramic = await buildCeramicNode(config.jest.services.ceramic, ipfs)
```

Comment:
This doesn't seem right.  This is going to make us always use an in-process `core` ceramic instance, regardless of test variant.  We only want the in-process node for the `local_node-private` variant.  The other two variants should use the http-client as they do today.  You need to decide whether to use the http-client or the local node based on the config that is passed in.

Reply:
Fixed.